src/webui/www/private/css/style.css

```diff
@@ -540,23 +540,29 @@ ul.filterList {
     padding-left: 0;
 }
 
+ul.filterList li:hover img,
+ul.filterList .selectedFilter img {
+    filter: var(--color-icon-hover);
```

Comment:
Yes, otherwise it would look like this:
![image](https://github.com/user-attachments/assets/bdd4d03c-f773-4827-96c9-020a4a6a7186)

How would you like it to be?

Reply:
Maybe use other filter? Like

```suggestion
    filter: drop-shadow(0 0 1px #000);
```

![screenshot](https://github.com/user-attachments/assets/fd14fa12-5083-4ab7-9a4c-12a4c321cd74)

Or more tricky

```suggestion
    filter: filter: drop-shadow(0 1px 0 #000) drop-shadow(0 -1px 0 #000) drop-shadow(1px 0 0 #000) drop-shadow(-1px 0 0 #000);
```

![screenshot](https://github.com/user-attachments/assets/b0e8f1a0-77e5-411c-a6a6-78ccac167635)

if we want it to be solid.

Reply:
@HanabishiRecca, I'll get to it before 5.1 releases for sure - I want to improve the dark theme in a way so such hover styles are not necessery (but first I need to up my styling game :d).